### PR TITLE
No Nunsense

### PR DESCRIPTION
**Pull request by NPC1314**
## About The Pull Request

Same as the earlier, new nun outfit, but with 100% more modularization
Selectable in loadout.
nunsense

## How This Contributes To The Skyrat Roleplay Experience

Improved nun outfit good.  Just want to play with prettier clothes, so trying to do the PR by the nunbers now,
Just want to play with it with minimum hassle, so keeping it as bare bones I can. 

## Changelog

:cl:
add: the nun outfit to loadout module
imageadd: new alternative nun outfit in chaplain module
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
again suggesting you replace the nunrobes upstream but either way its good

**Comment by NPC1314** [comment]
> again suggesting you replace the nunrobes upstream but either way its good

I appreciate the thought, but my experience is trying to replace or remove ANYTHING on public databases is just a massive headache and greatly increases the risk of the PR getting stuck. So to save myself grief and wasted work I prefer to just add stuff, and since I don't play TG upstream it makes sense for me to put the module here and be done with it.

**Comment by YakumoChen** [comment]
> trying to replace or remove ANYTHING on public databases is just a massive headache

 it's literally just replacing a sprite in a .dmi file upstream, no entire modular folder or code

Also something something "follow modularisation guidelines exactly"

File: modular_skyrat/modules/chaplain/code/chaplainmodule.dm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// FILE FOR CHAPLAIN RELATED SKYRAT CODE
+
+// Nun outfit
+/obj/item/clothing/head/nun
+	name = "nun headdress"
+	desc = "A cloth headpiece that covers the hair and ears. The iconic uniform of nuns across the galaxy."
+	icon = 'modular_skyrat/modules/chaplain/icons/nun.dmi'
+	worn_icon = 'modular_skyrat/modules/chaplain/icons/nunmob.dmi'
```
Comment (ghost) [inline modular_skyrat/modules/chaplain/code/chaplainmodule.dm:8]: Clothing icons should be going into modular_skyrat/master_files/icons/(obj or mob)/clothing, please!

File: modular_skyrat/modules/chaplain/code/chaplainmodule.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	icon_state = "nun_head"
+	inhand_icon_state = "cowboy_hat_black"
+	flags_inv = HIDEHAIR | HIDEEARS
+
+/obj/item/clothing/under/nunrobes
+	name = "nun robes"
+	desc = "Plain black robes in a feminine style, meant to display piety and turn thoughts away from sin."
+	icon = 'modular_skyrat/modules/chaplain/icons/nun.dmi'
+	worn_icon = 'modular_skyrat/modules/chaplain/icons/nunmob.dmi'
```
Comment (ghost) [inline modular_skyrat/modules/chaplain/code/chaplainmodule.dm:17]: This one in particular should be put into the 'civilian' icon file in aforementioned master files, OR made into a suit item (and put into the suit icon masterfile still) - Wearing the robes over the uniform tends to be the choice for chaplain outfits, especially when they're as long as these ones.

**Comment by ghost** [comment]
The icons themselves leave a bit to be desired, specifically the shading in areas like this one feeling a bit randomized -
image
but overall I quite enjoy it. Though, I second the idea of simply replacing TGs: you won't have to do any code, only changing icons in the .dmi's, and they'll be able to help explain some ways of improving the sprites

**Comment by softcerv** [comment]
> I prefer to just add stuff, and since I don't play TG upstream it makes sense for me to put the module here and be done with it.

image
if you can make a PR upstream, you should do it. It lets more people benefit in the end.

**Pull request closed without merging**